wordpress-mobile/WordPress-iOS
# Blogging Prompts List: initial view

Ref: #18502

This adds an initial Blogging Prompts list view.

To test:
- Enable `bloggingPrompts` feature flag.
- Select the My Site `Home` tab.
- On the prompt card menu, select `View more prompts`.
- Verify the `Prompts` view is displayed.
  - 10 stub empty rows.
  -  Filter tab bar at the top under the nav bar.


https://user-images.githubusercontent.com/1816888/166591274-2bacf16c-617b-4e48-b164-57755f07fde2.mp4



## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
